core/pom.xml

```diff
@@ -106,11 +106,6 @@ THE SOFTWARE.
       <artifactId>jna-posix</artifactId>
       <version>1.0.3-jenkins-1</version>
     </dependency>
-    <dependency>
-      <groupId>com.github.jnr</groupId>
-      <artifactId>jnr-posix</artifactId>
```

Comment:
There are usages outside the core: https://github.com/jenkinsci/ssh-agent-plugin/blob/fd42f5e7aeb3772e1baf264cc1b39761dcda79a0/src/main/java/com/cloudbees/jenkins/plugins/sshagent/jna/AgentServer.java